main.py

```diff
@@ -0,0 +1,48 @@
+from BACnetAdapter import BACnetAdapter
+
+from bacpypes.core import run, enable_sleeping
+from bacpypes.service.device import LocalDeviceObject
+
+import sys, argparse
+
+credentials = {}
+
+adapter_version = "1.0"
+
+print "BACnet-Adapter v" + adapter_version
+
+
+def _parse_args(argv):
+    parser = argparse.ArgumentParser(description='BACnetAdapter')
+    parser.add_argument('--systemKey', required=True, help='The System Key of the ClearBlade platform the BACnet adapter will connect to.')
+    parser.add_argument('--systemSecret', required=True, help='The System Secret of the ClearBlade platform the BACnet adapter will connect to.')
+    parser.add_argument('--deviceName', required=True, help='The name of the device, defined within the devices table of the ClearBlade platform, representing the BACnet Adapter.')
+    parser.add_argument('--activeKey', required=True, help='The Active Key, defined within the devices table of the ClearBlade platform, corresponding to the BACnet Adapter.')
+    parser.add_argument('--ipAddress', required=True, help='The ip address of the device this adapter is running on')
```

Comment:
Nope, looks like it has to be an IP for bacpypes to work. Added some text that says that specifically